src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -101,13 +102,13 @@ public Content processPreview(EomFile eomFile, String transactionId) {
         UUID uuid = UUID.fromString(eomFile.getUuid());
 
         PublishEligibilityChecker eligibilityChecker =
-            PublishEligibilityChecker.forEomFile(eomFile, uuid, transactionId);
+                PublishEligibilityChecker.forEomFile(eomFile, uuid, transactionId);
```

Comment:
I don't really like it... It does not seem to have a max characters per line limit.
The only decent thing is the way that it formats method parameters.